packages/app-accounts/src/Forgetting.tsx

```diff
@@ -57,20 +61,31 @@ class Forgetting extends React.PureComponent<Props> {
           <Button
             isPrimary
             onClick={doForget}
-            text='Delete'
+            text={t('forget.delete', {
+              defaultValue: 'Delete'
```

Comment:
I should have picked this up before - can we call this "Forget" rather. Basically "Delete" says that is is removed from the chain, rather in this case the account and associated balances will still be there, but this instance just "forgot" about it.

Reply:
Same with the "... you want to delete ..." below, should be "... remove from this browser...", or keeping with "forget".